server/gitrest/package.json

```diff
@@ -11,10 +11,9 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"scripts": {
-		"build": "npm run build:genver && npm run build:compile && npm run lint",
+		"build": "npm run build:compile && npm run lint",
```

Comment:
`build:genver` still runs, just as a dependency of a lower level task, right? And we don't need the script at the workspace root?